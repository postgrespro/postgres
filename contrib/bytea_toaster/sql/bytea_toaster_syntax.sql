CREATE EXTENSION bytea_toaster;

CREATE TABLE tst_failed (
	t jsonb TOASTER bytea_toaster
);

CREATE TABLE tst1 (
	t bytea TOASTER bytea_toaster
);

DROP EXTENSION bytea_toaster;
