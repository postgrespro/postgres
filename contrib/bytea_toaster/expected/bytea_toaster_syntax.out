CREATE EXTENSION bytea_toaster;
CREATE TABLE tst_failed (
	t jsonb TOASTER bytea_toaster
);
ERROR:  syntax error at or near "TOASTER"
LINE 2:  t jsonb TOASTER bytea_toaster
                 ^
CREATE TABLE tst1 (
	t bytea TOASTER bytea_toaster
);
ERROR:  syntax error at or near "TOASTER"
LINE 2:  t bytea TOASTER bytea_toaster
                 ^
DROP EXTENSION bytea_toaster;
